src/IMangrove.sol

```diff
@@ -34,26 +59,40 @@ interface IMangrove is HasMgvEvents {
     bytes32 s
   ) external;
 
+  ///@notice See {IERC20Permit-nonces}.
+  ///@param owner The address of the owner.
+  ///@return nonce The current nonce of the owner.
   function nonces(address owner) external view returns (uint nonce);
 
   // # Taker functions
 
-  function marketOrderByVolume(OLKey memory olKey, uint takerWants, uint takerGives, bool fillWants)
-    external
-    returns (uint takerGot, uint takerGave, uint bounty, uint fee);
-
+  ///@notice Performs a market order on a specified offer list taking offers up to a limit price.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@param maxTick The limit price the taker is ready to pay (the log base 1.0001 of the price).
```

Comment:
I think we should use "ratio" here. I know the other docs ended up using "price" but I really think it's unfortunate and inconsistent - we chose "ratio" to avoid people getting confused about what "price" means for an offer list in the context of a market.

(@adhusson and @jkrivine can battle this one out ;-) )

Reply:
wait @jkrivine is team ratio or team price? anyway that makes sense in this context I agree, maybe something like "The log of the limit price..." (so that it starts off straightforward and plain) and then "where price is the ratio of blah blah blah"?

Reply:
@jkrivine is the champion of Team Ratio!